dbms/programs/server/MySQLHandler.cpp

```diff
@@ -266,29 +267,60 @@ void MySQLHandler::comPing()
     packet_sender->sendPacket(OK_Packet(0x0, client_capability_flags, 0, 0, 0), true);
 }
 
-void MySQLHandler::comQuery(ReadBuffer & payload)
+static bool isFederatedServerSetupCommand(String query)
 {
-    bool with_output = false;
-    std::function<void(const String &)> set_content_type = [&with_output](const String &) -> void {
-        with_output = true;
-    };
+    if ((0 == strncasecmp("SET NAMES", query.c_str(), 9)) || (0 == strncasecmp("SET character_set_results", query.c_str(), 25))
```

Comment:
`if` is redundant, it can be just a `return` statement.
`return (0 == strncasecmp("SET NAMES", query.c_str(), 9)) || ...`

Reply:
done